pkg/operation/botanist/component/vpnseedserver/vpn_seed_server.go

```diff
@@ -819,6 +819,9 @@ var envoyConfig = `static_resources:
   clusters:
   - name: dynamic_forward_proxy_cluster
     connect_timeout: 20s
+    circuitBreakers:
+        thresholds:
+        - maxConnections: 8192
```

Comment:
Shouldn't this be indented as

```suggestion
    circuitBreakers:
      thresholds:
      - maxConnections: 8192
```

?

Reply:
Agreed.